runtime/onert/core/include/backend/ITensorBuilder.h

```diff
@@ -70,10 +70,7 @@ struct ITensorBuilder
    *
    * @note   Backend should implement this when it has StaticTensorManager and DynamicTensorManager
```

Comment:
Well, I'm thinking about refactoring that all backends have `ITensorRegistry` and remove duplicated methods between `ITensorRegistry` and `ITensorBuilder` - #3179.

Reply:
since acl has `TensorRegistry()` (which returns nullptr) but it does not have `StaticTensorManager` and `DynamicTensorManager`, I think this like should be deleted. :-)